### PR TITLE
Fix tokio blocking 'not currently running on the Tokio runtime.' panic

### PR DESCRIPTION
**Pull request by expenses**
@montekki

### CODE REVIEW

**Comment by expenses** [comment]
From: https://github.com/paritytech/polkadot/pull/679#issuecomment-564563073

File: validation/Cargo.toml [hunk 1]
```diff
@@ -12,6 +12,7 @@ tokio = { version = "0.2.4", features = ["rt-core", "blocking"] }
 derive_more = "0.14.1"
 log = "0.4.8"
 exit-future = "0.2.0"
+tokio-executor = { version = "0.2.0-alpha.6", features = ["blocking"] }
```
Comment (montekki) [inline validation/Cargo.toml]: Uhh, an `alpha`? I hereby request the opinion of @tomaka on this one

**Comment by montekki** [comment]
Other than that, it looks like the nodes are not panicking any more, work, author blocks and do stuff in the desired fashion.

File: validation/Cargo.toml (hunk 1 above)
Comment (expenses) [reply validation/Cargo.toml]: That's what it was before #673.

Comment (expenses) [reply validation/Cargo.toml]: I could revert further, but if it works, it works 🤷‍♀

**Pull request merged**